### PR TITLE
Issue7 transform agent to sending json

### PR DESCRIPTION
**Pull request by Ilargi12**
https://github.com/agh-docker-monitoring/dockiera-agent/issues/7

### CODE REVIEW

File: src/agent/agent.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+from .config import BACKEND_ENDPOINT, HOME_PATH, HOST_ID, MAX_WORKERS, SLEEP_TIME
+from .dataclasses import BasicMetric, ContainerSummary, HostSummary
+
+
+def get_metric_from_data(metric_name: str, data: Any) -> BasicMetric:
+    if metric_name == "virtual_memory":
+        return BasicMetric(data.used, data.total, data.percent)
+    elif metric_name == "disk_memory":
+        return BasicMetric(data.used, data.total, data.percent)
```
Comment (jakubsolecki) [inline src/agent/agent.py:20]: Could be covered as one case

File: src/agent/mock_agent.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        self.influxdb_data_writer = self.influxdb_client.write_api(ASYNCHRONOUS)
+        self.docker_client = docker.from_env()
+        self.executor = ThreadPoolExecutor(max_workers=MAX_WORKERS)
+        self.containers_info = self.get_containers_info_from_csv()
+        self.total_virtual_memory = 4169125888
+        self.total_disk_memory = 125910429696
+        self.percent_virtual_memory = float("50")
+        self.percent_disk_memory = float("50")
+
+    def get_containers_info_from_csv(self):
```
Comment (jakubsolecki) [inline src/agent/mock_agent.py:26]: Do we have possibility to fetch actual data from Docker API or just from mocks at this moment?

File: src/agent/config-example.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#INFLUX
+INFLUXDB_ORGANIZATION = "agh-utc"
+INFLUXDB_URL = "http://localhost:8086"
+INFLUXDB_BUCKET = "dockeria"
+INFLUXDB_TOKEN = "yourtokenhere"
+INFLUXDB_WRITE_INTERVAL_TIME = 10
```
Comment (jakubsolecki) [inline src/agent/config-example.py:6]: Considering fact that Influx is now optional and accessed only from and by backend component, do we still need those variables, or at least named like that?

**Review by jakubsolecki — COMMENTED** [review]
Let @kraleppa see and check comments

**Comment by kraleppa** [comment]
Ehh

File: src/agent/mock_agent.py (hunk 2 above)
Comment (Ilargi12) [reply src/agent/mock_agent.py:26]: For now we have a possibility to only fetch actual data from Docker API.
The mock agent refactor will occur in following commits

File: src/agent/config-example.py (hunk 3 above)
Comment (Ilargi12) [reply src/agent/config-example.py:6]: Sure thang

**Comment by Ilargi12** [comment]
> Ehh
> 
> 

Nearly 700 lines are just packages dependencies.
Will fix that.

**Comment by Ilargi12** [comment]
Please take another look boys

**Review by jakubsolecki — APPROVED** [review]
Good job

**Pull request merged**